### PR TITLE
Update testing.rst

### PR DESCRIPTION
**Pull request by Yohann76**
this link is dead

### CODE REVIEW

**Review by HeahDude — COMMENTED** [review]
Thanks for that PR, I suggest another fix instead.

File: testing.rst [hunk 1]
```diff
@@ -484,7 +484,7 @@ script::
 AJAX Requests
 ~~~~~~~~~~~~~
 
-The Client provides a :method:`Symfony\\Component\\BrowserKit\\Client::xmlHttpRequest`
+The Client provides a `xmlHttpRequest`
```
Comment (HeahDude) [inline testing.rst]: The `Client` has been deprecated instead, we should replace it by ``:class:`Symfony\Bundle\FrameworkBundle\KernelBrowser` `` instead. But this should be done everywhere in this file.

File: testing.rst [hunk 2]
```diff
@@ -484,7 +484,7 @@ script::
 AJAX Requests
 ~~~~~~~~~~~~~
 
-The Client provides a :method:`Symfony\\Component\\BrowserKit\\Client::xmlHttpRequest`
+The Client provides a :class:`Symfony\\Bundle\\FrameworkBundle\\KernelBrowser :: xmlHttpRequest`
```
Comment (skmedix) [inline testing.rst]: This is no longer valid as there is no `xmlHttpRequest` method in `KernelBrowser `. This means that the code below is also incorrect and needs further changes.

Comment (skmedix) [inline testing.rst]:
```suggestion
The Client provides a :class:`Symfony\\Bundle\\FrameworkBundle\\KernelBrowser`
```

**Comment by wouterj** [comment]
Hi @Yohann76. I'm very sorry for completely missing this PR. While trying to merge this one, I discovered that someone already fixed this link (see https://symfony.com/doc/5.0/testing.html#ajax-requests ). So unfortunately, I cannot merge this PR :(

Thanks anyway for proposing fixes for invalid links! I hope future PRs of yours will get merged :)

**Pull request closed without merging**